saleor/tests/e2e/orders/utils/draft_order_create.py

```diff
@@ -9,41 +9,42 @@
       code
     }
     order {
-        id
-        created
-        discounts {
-            amount {
-                amount
-            }
+      id
+      created
+      discounts {
+        amount {
+          amount
         }
-        billingAddress {
+      }
+      billingAddress {
         streetAddress1
+      }
+      shippingAddress {
+        streetAddress1
+      }
+      isShippingRequired
+      shippingMethods {
+        id
+      }
+      lines {
+        productVariantId
+        quantity
+        undiscountedUnitPrice {
+          gross {
+            amount
+          }
         }
-        shippingAddress {
-            streetAddress1
-        }
-        shippingMethods {
-            id
+        unitPrice {
+          gross {
+            amount
+          }
         }
-        lines {
-            productVariantId
-            quantity
-            undiscountedUnitPrice {
-                gross {
-                amount
-                }
-            }
-            unitPrice {
-                gross {
-                    amount
-                }
-            }
-            totalPrice {
-                gross {
-                    amount
-                }
-            }
```

Comment:
I am not sure this should be removed - have you checked those fields are unused?

Reply:
I only added `isShippingRequired` and updated formatting (copied and pasted form GraphQL playground. If you take a look at the whole [file](https://github.com/saleor/saleor/blob/87ff6b933879e8f9e5b0e28b69fade1911e702a2/saleor/tests/e2e/orders/utils/draft_order_create.py) on this branch all fields are still here.